v21.2/architecture/overview.md

```diff
@@ -9,6 +9,10 @@ CockroachDB was designed to create the source-available database we would want t
 
 However, you definitely do not need to understand the underlying architecture to use CockroachDB. These pages give serious users and database enthusiasts a high-level framework to explain what's happening under the hood.
 
+{{site.data.alerts.callout_success}}
```

Comment:
We also should add a similar section to `cockroachcloud/architecture.md`.

Reply:
Done.